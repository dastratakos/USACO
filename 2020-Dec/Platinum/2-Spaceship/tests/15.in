20 20 20
11111111101101101000
11111111111011110111
11111001011110011011
11101111011001011010
11001110111011110110
00100011011111110111
00111000111110110000
01110101010111011100
10111110111101110001
00011011011011101111
11110010111111111010
11010110110110010111
01101010100111011011
10111101011011111011
01111011010110101101
10110011111100110110
10111100001100110101
10111110111011111000
11100101000001101010
10110001001110110101
4 8 1 18
4 16 2 17
2 13 1 15
4 9 2 9
2 18 5 15
3 10 1 5
4 5 4 9
4 10 11 11
12 19 5 5
5 5 2 20
2 19 18 9
11 14 3 4
2 2 1 4
2 9 5 6
3 13 4 6
2 15 3 8
3 3 1 18
5 10 1 8
2 16 2 15
16 17 16 17
