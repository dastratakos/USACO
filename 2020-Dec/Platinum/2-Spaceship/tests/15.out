956525923
885099666
653370559
609353986
768358511
784189269
441227644
348011264
915269384
412433449
704663660
476832915
386432757
420649503
652708650
611704168
398550395
950091228
369306610
717832324
